lib/mapper-v1.js

```diff
@@ -34,7 +35,25 @@ exports.identify = function(msg, settings) {
   ret.user_id = msg.userId();
   ret.custom_attributes = traits;
 
-  if (msg.created()) ret.remote_created_at = time(msg.created());
+  var created = time(msg.created())
+
+  if (created) {
```

Comment:
Yep, I'll leave notes describing the difference and links to the Intercom documentation, then open a new PR. Thanks!